themes/gruvbox-dark-hard.json

```diff
@@ -866,7 +866,7 @@
     "list.focusBackground": "#3c3836",
     "list.focusForeground": "#ebdbb2",
     "list.inactiveSelectionForeground": "#689d6a",
-    "list.inactiveSelectionBackground": "#1d202100",
+    "list.inactiveSelectionBackground": "#3c3836",
```

Comment:
we should use `#3c383680` instead, to be aligned with the active one, same comments with others, just need to append `80` to the value